src/openrct2/ride/coaster/meta/LIMLaunchedRollerCoaster.h

```diff
@@ -31,7 +31,9 @@ constexpr RideTypeDescriptor LIMLaunchedRollerCoasterRTD =
                      RIDE_TYPE_FLAG_ALLOW_REVERSED_TRAINS,
     .RideModes = EnumsToFlags(RideMode::PoweredLaunchPasstrough, RideMode::PoweredLaunch, RideMode::PoweredLaunchBlockSectioned),
     .DefaultMode = RideMode::PoweredLaunchPasstrough,
-    .OperatingSettings = { 10, 31, 26, 18, 52, 0 },
```

Comment:
This changes the maximum brakes speed, is that intentional?

Reply:
No and yes. No, this doesn't change maximum brake speed because the value was never used. Yes, it is intentional because it makes the maximum speed value match current behavior.